components/lock.rst

```diff
@@ -0,0 +1,257 @@
+.. index::
+   single: Lock
+   single: Components; Lock
+
+The Lock Component
+==================
+
+    The Lock Component creates and manages `locks`_, a mechanism to provide
+    exclusive access to a shared resource.
+
+.. versionadded:: 3.3
+    The Lock component was introduced in Symfony 3.3.
+
+Installation
+------------
+
+You can install the component in 2 different ways:
+
+* :doc:`Install it via Composer </components/using_components>` (``symfony/lock`` on `Packagist`_);
+* Use the official Git repository (https://github.com/symfony/lock).
+
+.. include:: /components/require_autoload.rst.inc
+
+Usage
+-----
+
+Locks are used to guarantee exclusive access to some shared resource. In
+Symfony applications, you can use locks for example to ensure that a command is
+not executed more than once at the same time (on the same or different servers).
+
+In order to manage the state of locks, you first need to create a ``Store``
```

Comment:
"you first need to create a ``Store``" => "a ``Store`` needs to be created first"